revpi_flat.c

```diff
@@ -162,6 +163,9 @@ static int revpi_flat_handle_ain(struct revpi_flat *flat)
 	ain_val *= REVPI_FLAT_AIN_CORRECTION;
 	ain_val = (int) div_s64(ain_val, 1000000000LL);
 
+	if (mode_current)
+		ain_val = div_s64(ain_val, 480);
```

Comment:
Is there a place where it is documented why this is done? If yes it would be great to have this linked here or a comment how this value was obtained.